### PR TITLE
feat(processor): Count Logs as Metric

### PR DESCRIPTION
**Pull request by antonblock**
### Proposed Change
- Add `count_logs` processor wrapping `logcountprocessor` in the collector
- Add documentation for the processor
- Update input parameter components to show documentation when present
- Update `evalComponents` to try and include a builtin `route` receiver

##### Checklist
- [ ] Changes are tested
- [ ] CI has passed

### CODE REVIEW

File: docs/www/integrations/processors/count_logs.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+## Count Logs Processor
+
+The Count Logs Processor can count the number of logs matching some filter, and create a metric with that value.
+Both the name and units of the created metric can be configured. Additionally, fields from matching logs can be preserved as metric attributes.
+
+## Supported Types
+
+| Metrics | Logs | Traces |
+| :--- | :--- | :--- |
+| | ✓ | |
```
Comment (jsirianni) [inline docs/www/integrations/processors/count_logs.md:18]: Im not sure how I feel about this. It operates within a log pipeline, but emits metrics. To me, supported types should be the pipelines the processor can run within. What do you think?

File: resources/source-types/route.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  version: 0.0.1
+  supported_platforms:
+    - macos
+    - linux
+    - windows
+  parameters: []
+
+  metrics:
+    receivers: |
+      - route:
```
Comment (jsirianni) [inline resources/source-types/route.yaml:18]: Missing newline

File: docs/www/integrations/processors/count_logs.md (hunk 1 above)
Comment (antonblock) [reply docs/www/integrations/processors/count_logs.md:18]: My reasoning was that `Supported Types` meant telemetry types that the processor can operate on. FWIW the processor doesn't exist in a metrics pipeline when rendered, just magically emits to the `route` receiver. I'm Ok marking both metrics and logs as supported, though, I think the docs and UI are clear enough about this only counting logs

File: docs/www/integrations/processors/filter_severity.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ The Severity Filter processor can be used to filter out logs that do not meet a
 | :---       | :---    | :---     | :--- |
 | severity   | `enum`  | `TRACE` | Minimum severity to match. Log entries with lower severities will be filtered. |
 
-Valud severity levels:
+Valued severity levels:
```
Comment (andykellr) [inline docs/www/integrations/processors/filter_severity.md:25]:
```suggestion
Valid severity levels:
```
?

Comment (antonblock) [reply docs/www/integrations/processors/filter_severity.md:25]: 🙁

**Review by andykellr — APPROVED** [review]

**Pull request merged**